Tests/SwiftDocCTests/Test Resources/TestDoxygenDiagnostics.symbols.json

```diff
@@ -0,0 +1,1463 @@
+{
+  "metadata" : {
+    "formatVersion" : {
+      "major" : 0,
+      "minor" : 5,
+      "patch" : 0
+    },
+    "generator" : "SymbolKit"
+  },
+  "module" : {
+    "name" : "Test",
+    "platform" : {
+      "architecture" : "x86_64",
+      "operatingSystem" : {
+        "minimumVersion" : {
+          "major" : 13,
+          "minor" : 0,
+          "patch" : 0
+        },
+        "name" : "macos"
+      },
+      "vendor" : "apple"
+    }
+  },
+  "relationships" : [
+    {
+      "kind" : "memberOf",
+      "source" : "c:@E@TestExtensions@TestExtensionsAutolinkedURLs",
+      "target" : "c:@E@TestExtensions",
+      "targetFallback" : null
+    },
+    {
+      "kind" : "memberOf",
+      "source" : "c:@E@TestExtensions@TestExtensionsFencedCodeBlocks",
+      "target" : "c:@E@TestExtensions",
+      "targetFallback" : null
+    },
+    {
+      "kind" : "memberOf",
+      "source" : "c:@E@TestExtensions@TestExtensionsGitHubFlavored",
+      "target" : "c:@E@TestExtensions",
+      "targetFallback" : null
+    },
+    {
+      "kind" : "memberOf",
+      "source" : "c:@E@TestExtensions@TestExtensionsHardNewlines",
+      "target" : "c:@E@TestExtensions",
+      "targetFallback" : null
+    },
+    {
+      "kind" : "memberOf",
+      "source" : "c:@E@TestExtensions@TestExtensionsNone",
+      "target" : "c:@E@TestExtensions",
+      "targetFallback" : null
+    },
+    {
+      "kind" : "memberOf",
+      "source" : "c:@E@TestExtensions@TestExtensionsStrikethroughs",
+      "target" : "c:@E@TestExtensions",
+      "targetFallback" : null
+    },
+    {
+      "kind" : "memberOf",
+      "source" : "c:@E@TestExtensions@TestExtensionsTables",
+      "target" : "c:@E@TestExtensions",
+      "targetFallback" : null
+    },
+    {
+      "kind" : "memberOf",
+      "source" : "c:@E@TestExtensions@TestExtensionsUnderscoresInWords",
+      "target" : "c:@E@TestExtensions",
+      "targetFallback" : null
+    },
+    {
+      "kind" : "inheritsFrom",
+      "source" : "c:objc(cs)Test",
+      "target" : "c:objc(cs)NSObject",
+      "targetFallback" : "NSObject"
+    },
+    {
+      "kind" : "memberOf",
+      "source" : "c:objc(cs)Test(cm)HTMLStringWithMarkdown:error:",
```

Comment:
I think it would be better to replace this test data with something more generic.

Reply:
Yeah +1 on this. If we can just generate a symbol graph from a test project with a single symbol I think it would be more clear what we're testing on here.